telethon/client/updates.py

```diff
@@ -14,6 +14,12 @@ class UpdateMethods(UserMethods):
 
     # region Public methods
 
+    def run_loop(self):
```

Comment:
Should we rename this for `idle()` to be consistent with the synchronous version? After all we're `idle()`'ing here until we're disconnected.

Reply:
I'm renaming it as `idle()`. `run_loop` is not quite self-explanatory at first glance in my opinion, what loop is it running and what does that loop? Meanwhile is like "stay idle, do nothing but wait".

Reply:
Or not I just don't know.

Reply:
I think `run_until_disconnected` would be the best name here, and then rename `connection_dropped` to `disconnected`, and the API will be consistent.

Reply:
But then I can't edit more than one file from GitHub and I have to clone it. Well then.